aviary/mission/gasp_based/phases/accel_phase.py

```diff
@@ -1,104 +1,208 @@
+            units="s",
+            duration_ref=duration_ref,
+        )
+
+        # States
+        phase.add_state(
+            "TAS",
+            fix_initial=fix_initial,
+            fix_final=False,
+            lower=TAS_lower,
+            upper=TAS_upper,
+            units="kn",
+            rate_source="TAS_rate",
+            targets="TAS",
+            ref=TAS_ref,
+            ref0=TAS_ref0,
+            defect_ref=TAS_defect_ref,
+        )
+
+        phase.add_state(
+            Dynamic.Mission.MASS,
+            fix_initial=fix_initial,
+            fix_final=False,
+            lower=mass_lower,
+            upper=mass_upper,
+            units="lbm",
+            rate_source=Dynamic.Mission.FUEL_FLOW_RATE_NEGATIVE_TOTAL,
+            targets=Dynamic.Mission.MASS,
+            ref=mass_ref,
+            ref0=mass_ref0,
+            defect_ref=mass_defect_ref,
+        )
+
+        phase.add_state(
+            Dynamic.Mission.DISTANCE,
+            fix_initial=fix_initial,
+            fix_final=False,
+            lower=distance_lower,
+            upper=distance_upper,
+            units="NM",
+            rate_source="distance_rate",
+            ref=distance_ref,
+            ref0=distance_ref0,
+            defect_ref=distance_defect_ref,
+        )
+
+        # Boundary Constraints
+        phase.add_boundary_constraint(
+            "EAS", loc="final", equals=EAS_constraint_eq, units="kn", ref=EAS_constraint_eq
+        )
+
+        phase.add_parameter(Dynamic.Mission.ALTITUDE, opt=False, units="ft", val=alt)
+
+        # Timeseries Outputs
+        phase.add_timeseries_output("EAS", output_name="EAS", units="kn")
+        phase.add_timeseries_output(
+            Dynamic.Mission.MACH, output_name=Dynamic.Mission.MACH, units="unitless")
+        phase.add_timeseries_output("alpha", output_name="alpha", units="deg")
+        phase.add_timeseries_output("aero.CL", output_name="CL", units="unitless")
+        phase.add_timeseries_output(
+            Dynamic.Mission.THRUST_TOTAL, output_name=Dynamic.Mission.THRUST_TOTAL, units="lbf")
+        phase.add_timeseries_output("aero.CD", output_name="CD", units="unitless")
+
+        return phase
+
+
+# Adding metadata for the AccelPhase
+AccelPhase._add_meta_data('fix_initial', val=False)
+AccelPhase._add_meta_data('EAS_constraint_eq', val=250, units='kn')
+AccelPhase._add_meta_data('time_initial_bounds', val=(0, 0), units='s')
+AccelPhase._add_meta_data('duration_bounds', val=(0, 0), units='s')
+AccelPhase._add_meta_data('duration_ref', val=1, units='s')
+AccelPhase._add_meta_data('TAS_lower', val=0, units='kn')
+AccelPhase._add_meta_data('TAS_upper', val=0, units='kn')
+AccelPhase._add_meta_data('TAS_ref', val=1, units='kn')
+AccelPhase._add_meta_data('TAS_ref0', val=0, units='kn')
+AccelPhase._add_meta_data('TAS_defect_ref', val=None, units='kn')
+AccelPhase._add_meta_data('mass_lower', val=0, units='lbm')
+AccelPhase._add_meta_data('mass_upper', val=0, units='lbm')
```

Comment:
If the user doesn't set a mass_upper in their phase_info, does it default to 0 as written here?
Or is this a case where there are other defaults that take precedence over this. (And if that is the case, it ties in with some of the confusion that Joerg talked about.)

This applies to all of the phases.

Reply:
Yep, you are correct. I retained all the default behavior from Aviary as-is, like shown here: https://github.com/OpenMDAO/om-Aviary/blob/main/aviary/mission/gasp_based/phases/accel_phase.py#L22

I made an issue for this to tackle in a subsequent PR. Let me know if you want a change for this PR: https://github.com/OpenMDAO/om-Aviary/issues/107